Formula/par2.rb

```diff
@@ -11,8 +11,16 @@ class Par2 < Formula
     sha256 "7257b39640dcf1894c2329129406a573b04dd263bf3b06283dc854b5ed17cf8e" => :el_capitan
   end
 
+  option "with-openmp", "Compile with OpenMP support"
+  needs :openmp if build.with? "openmp"
+
   def install
-    system "./configure", "--prefix=#{prefix}"
+    if build.with? "openmp"
+      ENV["CXX"] = "#{Formula["llvm"].opt_bin}/clang++"
```

Comment:
No, we support OpenMP in other ways, such as using GCC as a compiler. So this formula should not hardcode anything specific to llvm.

The configure script uses `AC_OPENMP`, so OpenMP should be automatically detected and used if available.